plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/EJBMediatorPropertiesEditionPartForm.java

```diff
@@ -175,20 +175,19 @@ public Composite createFigure(final Composite parent, final FormToolkit widgetFa
 	public void createControls(final FormToolkit widgetFactory, Composite view) {
 		CompositionSequence eJBMediatorStep = new BindingCompositionSequence(propertiesEditionComponent);
 		CompositionStep propertiesStep = eJBMediatorStep.addStep(EsbViewsRepository.EJBMediator.Properties.class);
-		propertiesStep.addStep(EsbViewsRepository.EJBMediator.Properties.description);
 		propertiesStep.addStep(EsbViewsRepository.EJBMediator.Properties.commentsList);
 		propertiesStep.addStep(EsbViewsRepository.EJBMediator.Properties.reverse);
 		propertiesStep.addStep(EsbViewsRepository.EJBMediator.Properties.beanstalk);
 		propertiesStep.addStep(EsbViewsRepository.EJBMediator.Properties.class_);
 		propertiesStep.addStep(EsbViewsRepository.EJBMediator.Properties.method);
-		propertiesStep.addStep(EsbViewsRepository.EJBMediator.Properties.sessionIdType);
-		propertiesStep.addStep(EsbViewsRepository.EJBMediator.Properties.sessionIdLiteral);
 		propertiesStep.addStep(EsbViewsRepository.EJBMediator.Properties.remove);
 		propertiesStep.addStep(EsbViewsRepository.EJBMediator.Properties.target);
 		propertiesStep.addStep(EsbViewsRepository.EJBMediator.Properties.jNDIName);
 		propertiesStep.addStep(EsbViewsRepository.EJBMediator.Properties.methodArguments);
-		propertiesStep.addStep(EsbViewsRepository.EJBMediator.Properties.sessionIdExpression);
-		
+                propertiesStep.addStep(EsbViewsRepository.EJBMediator.Properties.sessionIdType);
```

Comment:
Formatting issue?

Reply:
This is due to the Eclipse formatting issue. If we open in the Eclipse IDE it will come in a formatted manner.